### DISCUSSION TITLE
Can starship output the default symbol for a given module?

### DISCUSSION THREAD

**Discussion by halloleo** [starship/starship#6019] [Q&A] [answered]
Can the `starship` command output the default symbol for a given module (or for all modules)?

(A bit like `starship module <MODULENAME>`, except it does output the prompt (or just the symbol) even if the current directory does not contain a matching `detect_` condition for \<MODULENAME\>.)

**Background:**

I want to create with a (shell) script a starship config that set each module's  symbol to the default symbol *without* the trailing space. I thought I could use 
`starship module -h` to get the names of all modules, then retrieve in a loop somehow for each module the default symbol and write this info formatted a toml to a file.

**Comment by davidkna** [comment, marked as answer]
`starship print-config` can be used to print f the computed (or default with `-d`) config. That is plain `starship print-config` or something like `starship print-config aws.symbol azure.symbol`.
